script/hassfest/mypy_config.py

```diff
@@ -137,6 +137,7 @@
     "homeassistant.components.nest.*",
     "homeassistant.components.netatmo.*",
     "homeassistant.components.netio.*",
+    "homeassistant.components.news",
```

Comment:
Please don't add anything new here. And because this integration is supposed to be internal, I think it should be added to `.strict-typing`.

Reply:
No integration that uses WS can have strict types.

Reply:
It's better to disable those warnings locally so we make sure the rest is typed strictly.

Reply:
Typing is in place to help the developer, which is not the case anymore... I wasted too much time trying to please it, and then again when trying to find a way to shut it up. mypy is too strict especially since the codebase is not ready for it. Just see the insanely huge ignore file. If I was a new contributor I'm sure I would have given up completely...

Adding comments or other artifacts to the code should not be done just to please a linter... when we have to do that, we are doing something wrong... yes I'm quite pissed with how much time I wasted on this... 

This line disables it for the init file only, which only has typing issues with WebSockets. Fixing that for WebSockets is out of the scope for this PR. 

If this is not acceptable, just mark this PR as a draft until it can be removed, that way I can focus on other things.

Reply:
Typing is helpful to make sure you're using these types correctly. It's helping in a way tests help to protect the code from errors. They're also very helpful to people reading the code which is usually not someone who wrote it.

The ignore file is insanely huge because for a long time all errors were ignored for majority of the codebase and over time many broken type hints were added. We're working on cleaning this up. Once again, don't add anything new there and it'll be getting smaller. We should work towards eliminating that list completely.

Sometimes you need to add type ignore comments which means something is wrong. Yes, that means that something is not being typed correctly. You're always welcome to improve typing in parts which you're using if you want to remove these annoying comments.

It's not necessary to enable strict check right away though. So you can leave parts which are hard to annotate correctly untyped, that's fine. Or ask questions, here're many people who can help you to figure out how to do it.

Regarding websockets specifically, take a look at https://github.com/home-assistant/core/pull/50915, you can help with review and testing if you're interested.

Reply:
I've fixed the websocket command typing in this PR. It was just the order of decorators. We realized recently that our previous pattern of ordering the decorators was not compatible with the typing. So we just need to swap those around a bit.

See https://github.com/home-assistant/core/pull/50171#discussion_r627545109

We don't need to type new integrations strictly, but it's good to not add more to the ignore list. :+1: